### ISSUE TITLE
CJS Compatible build?

### ISSUE THREAD

**Issue by jrnail23** [derangeddk/serialize-every-error#7]
Hi @hypesystem, I found this package from your discussion in [pino-std-serializers](https://github.com/pinojs/pino-std-serializers/pull/110#issuecomment-1193083904), and I wanted to give it a try, as some of my team members really want the full error serialization you're looking to do here.  Unfortunately, our shared logger lib that we use this stuff with is a CommonJS package, so I'm currently unable to use this package.
Is there any chance you could add cjs compatibility?

**Comment by hypesystem** [comment]
Yeah, for sure! If you want to have a stab at it that would work for me, too :smile: Very open to PRs.

I'm not sure what the best approach is for supporting it. Can you give me some more details on your setup/needs and maybe a minimal example of where importing this package *doesn't* work? That would be great to work from.
